pandapipes/topology/create_graph.py

```diff
@@ -108,13 +108,14 @@ def create_nxgraph(net, include_pipes=True, respect_status_pipes=True,
     else:
         mg = nx.Graph()
 
-    branch_params = {k: v for k, v in kwargs.items() if any(
-        k.startswith(par) for par in ["include", "respect_status", "weighting"])}
+    branch_kw = ["include", "respect_status", "weighting"]
+    branch_params = {k: v for k, v in kwargs.items() if any(k.startswith(par) for par in branch_kw)}
     loc = locals()
-    branch_params.update({"%s_%s" % (par, bc): loc.get("%s_%s" % (par, bc))
-                          for par in ["include", "respect_status", "weighting"]
+    branch_params.update({"%s_%s" % (par, bc): loc.get("%s_%s" % (par, bc)) for par in branch_kw
                           for bc in ["pipes", "valves", "pumps", "press_controls",
-                                     "mass_circ_pumps", "press_circ_pumps", "valve_pipes"]})
+                                     "mass_circ_pumps", "press_circ_pumps", "valve_pipes",
+                                     "flow_controls"]})
+
```

Comment:
Yes, the purpose of the defined arguments of the function is more related to convenience and documentation. But the user can add arguments related to how to consider certain branch components in the kwargs, if they belong to the network model. As I forgot to add the "flow_control" component to the default branch_params update, they were not considered upon graph creation, because they are part of the given arguments (not kwargs), but not added in the branch_params construction.